### PR TITLE
Add unfurling for Twitter links

### PR DESCRIPTION
**Pull request by ismaeldm95**
Adds unfurling for Twitter links

Twitter previews can now be displayed in the chat view.

**Important:** Due to limitations in twitter's oEmbed API, thumbnailUrl cannot be retrieved without major changes in how we process link previews. That's why a change in status-react is needed in order to hide the image view when thumbnailUrl is empty.

<details>
  <summary>Expand to see screenshot. ⬇</summary>

twitter-unfurling

</details>

Closes status-react issue #11425
https://github.com/status-im/status-react/issues/11425

### CODE REVIEW

File: protocol/urls/urls.go [hunk 1]
```diff
@@ -148,6 +161,46 @@ func GetYoutubePreviewData(link string) (previewData LinkPreviewData, err error)
...
+func GetTwitterOembed(url string) (data TwitterOembedData, err error) {
+	oembedLink := fmt.Sprintf(TwitterOembedLink, url)
+	jsonBytes, err := GetURLContent(oembedLink)
+	if err != nil {
+		return data, fmt.Errorf("can't get bytes from twitter oembed response on %s link", oembedLink)
+	}
+
+	err = json.Unmarshal(jsonBytes, &data)
+	if err != nil {
+		return data, fmt.Errorf("can't unmarshall json")
```
Comment (cammellos) [inline protocol/urls/urls.go]: You might want to wrap the original error, something like:

```
return data, fmt.Errorf("can't unmarshall json: %w", err)
```
Similarly a few lines below

https://blog.golang.org/go1.13-errors#TOC_3.3.

**Review by cammellos — APPROVED** [review]
Looks good, thanks for the contribution!

There's a few linting issues:

```

[2021-08-03T21:57:52.020Z] protocol/urls/urls.go:25:2: struct field `Html` should be `HTML` (golint)

[2021-08-03T21:57:52.020Z] 	Html         string `json:"html"`

[2021-08-03T21:57:52.020Z] 	^

[2021-08-03T21:57:52.020Z] protocol/urls/utils.go:16:6: func stripHtmlTags should be stripHTMLTags (golint)

[2021-08-03T21:57:52.020Z] func stripHtmlTags(s string) string {

```

The linter wants fully capitalized `HTML`, rather than camelCase

**Review by Samyoul — APPROVED** [review]
Apart from the comment from @cammellos this is a nice implementation. Thank you for you work.

**Comment by ismaeldm95** [comment]
@cammellos @Samyoul thanks a lot for the feedback! I'll fix this asap.

**Comment by flexsurfer** [comment]
@ismaeldm95 could you please resolve version conflict ,and we'll merge it

**Comment by ismaeldm95** [comment]
> @ismaeldm95 could you please resolve version conflict ,and we'll merge it

Done! I bumped the patch version. Let me know if there's anything else I need to change 💪💪

**Pull request merged**